package.json

```diff
@@ -162,10 +162,10 @@
 		"prettier": "npm:wp-prettier@1.19.1",
 		"progress": "2.0.3",
 		"puppeteer": "npm:puppeteer-core@3.0.0",
-		"react": "16.9.0",
-		"react-dom": "16.9.0",
-		"react-native": "0.61.5",
-		"react-test-renderer": "16.9.0",
+		"react": "16.11.0",
```

Comment:
Hey @gziolo @youknowriad any concern, if we bump, react to `16.11.0` in this PR?

Reply:
It’s fine 👍